release/_package.sh

```diff
@@ -33,20 +33,21 @@ function generate_release_properties_file {
 
 	local bundle_file_name="liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-tomcat-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.7z"
 
-	local product_version="DXP ${_PRODUCT_VERSION^^}"
-
-	product_version="${product_version/-/ }"
-
+	local product_version="${_PRODUCT_VERSION^^}"
 	local target_platform_version="${_PRODUCT_VERSION}"
 
 	if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "dxp" ]
 	then
-		target_platform_version=$(echo ${target_platform_version} | sed -r 's/-u/.u/')
+		product_version="DXP ${_PRODUCT_VERSION^^}"
```

Comment:
@Wykenz, since we're already setting the value `_PRODUCT_VERSION^^` to the variable `product_version`, why not just add the prefixes `DXP` and `Portal` to the content of the variable `product_version` as mentioned in https://github.com/natocesarrego/liferay-docker/pull/95#discussion_r1664052766?